rdkit/meta.yaml

```diff
@@ -1,13 +1,17 @@
+{% set data = load_setup_py_data(setup_file='pkg_version.py', permit_undefined_jinja=False) %}
+
 package:
   name: rdkit
-  version: "__conda_version__"
+  version: "{{ data.get('rdkitVersion') }}"
 
 source:
-  git_url: https://github.com/rdkit/rdkit.git
-  git_rev: Release_2017_03
+  git_url: https://github.com/ptosco/rdkit.git
```

Comment:
same here, of course.

Reply:
Hi Riccardo, I have left those paths in to enable testing while the changes in the rdkit trunk are not yet merged into master - I'll revert to the usual paths once the rdkit PR is accepted.

Reply:
@ptosco @rvianello For various reasons I prefer the versioning method seen in #59, mainly because the versioning is automatic.  If you don't have any complaints, I could do a PR against this one.

Reply:
I don't mind, I made this PR just to restore a working conda build while keeping the same versioning tag as with `__conda_version__`, but of course I'm happy with any versioning mechanism.

Reply:
I do not have any strong preferences, I like the general approach in #59 because it tends to reduce the amount of code we may need to maintain, I wasn't sure if it could work with how tags are formatted or positioned on the various branches in the rdkit repository. This PR on the other hand also looked fine to me as it didn't introduce any change apart the fix.